scripts/generate-cluster.sh

```diff
@@ -69,8 +73,8 @@ PROJECT_ID=${PACKET_PROJECT_ID}
 FACILITY=${PACKET_FACILITY}
 
 # and now export them all so envsubst can use them
-export PROJECT_ID FACILITY NODE_OS WORKER_NODE_TYPE MASTER_NODE_TYPE POD_CIDR SERVICE_CIDR SSH_KEY KUBERNETES_VERSION
-${CLUSTERCTL} ${CONFIG_OPT} config cluster ${CLUSTER_NAME} > $TEMPLATE_OUT
+export PROJECT_ID FACILITY NODE_OS WORKER_NODE_TYPE MASTER_NODE_TYPE POD_CIDR SERVICE_CIDR SSH_KEY KUBERNETES_VERSION WORKER_MACHINE_COUNT CONTROL_PLANE_MACHINE_COUNT
+${CLUSTERCTL} ${CONFIG_OPT} config cluster ${CLUSTER_NAME} --from file://$PWD/templates/cluster-template.yaml > $TEMPLATE_OUT
```

Comment:
Why is this from a file? Shouldn't it be using the official one?

Reply:
It depends what's your point of view. I run `make cluster` when I am developing and if I change the template locally I would like to use that.

A user that does not checkout this project will use `clusterctl` and not `make`.

A user that will checkout the project only to run the `make cluster` will get what it checks out.

To me this is a development utility more than something a user will use

Reply:
OK, got it. Makes sense.